src/screens/TripDetails/DepartureDetails/index.tsx

```diff
@@ -148,14 +149,17 @@ export default function DepartureDetails({navigation, route}: Props) {
 
           {isTicketingEnabledAndWeCantSellTicketForDeparture && (
             <MessageBox
-              containerStyle={styles.ticketMessage}
-              type="warning"
-              message={
-                t(DepartureDetailsTexts.messages.ticketsWeDontSell) +
-                (someLegsAreByTrain
-                  ? `\n\n` + t(DepartureDetailsTexts.messages.collabTicketInfo)
-                  : ``)
-              }
+              containerStyle={styles.messageBox}
+              type="info"
+              message={t(DepartureDetailsTexts.messages.ticketsWeDontSell)}
+            />
+          )}
+
+          {someLegsAreByTrain && (
+            <MessageBox
+              containerStyle={styles.messageBox}
+              type="info"
+              message={t(DepartureDetailsTexts.messages.collabTicketInfo)}
             />
           )}
```

Comment:
Just a heads up @ingvisim: Looks like this will create conflicts with what you have been working on. Probably more places as well, like in `TripMessages`.